packages/tools/storybook/stories/components/chip/chip.specifications.stories.mdx

```diff
@@ -2,7 +2,7 @@ import { Meta } from '@storybook/addon-docs';
 import SpecificationsChip from '@ovhcloud/ods-specifications/components/chip/specifications-chip.mdx';
 import Notes from '../notes.mdx';
 
-<Meta title="UI Components/Chip [atom]/Specifications" />
+<Meta title="UI Components/Chip [new]/Specifications" />
```

Comment:
would be best to keep both info don't you think?
<img width="173" alt="Capture d’écran 2023-06-30 à 16 18 25" src="https://github.com/ovh/design-system/assets/1390934/34f5a767-f52a-4502-bee4-882879c81e00">